src/alert.cpp

```diff
@@ -27,6 +27,8 @@ using namespace std;
 map<uint256, CAlert> mapAlerts;
 CCriticalSection cs_mapAlerts;
 
+static void Notify(const std::string& strMessage, bool fThread);
```

Comment:
This declaration could be saved if the the definition of the newly created method was moved below the definition of the static function.